core/ui/EditorToolbar/preview-type-dropdown.tid

```diff
@@ -1,33 +1,55 @@
-title: $:/core/ui/EditorToolbar/preview-type-dropdown
```

Comment:
Sorry @duarteframos, I'm afraid you're not at all doing the right thing here? To me it seems like you're overwriting another component rather than modifying the one for images, e.g. **$:/core/ui/EditorToolbar/picture-dropdown**?!?